tests/tasks/test_default_assets.py

```diff
@@ -143,15 +143,15 @@ async def test_vector_source_asset(batch_client, async_client):
                     "field_name": "gfw_geojson",
                     "field_alias": "gfw_geojson",
                     "field_description": None,
-                    "field_type": "character varying",
+                    "field_type": "text",
                     "is_feature_info": False,
                     "is_filter": False,
                 },
                 {
                     "field_name": "gfw_bbox",
                     "field_alias": "gfw_bbox",
                     "field_description": None,
-                    "field_type": "geometry",
+                    "field_type": "ARRAY",
```

Comment:
Does the current implementation return the type as Array? I might need to change this later to something like double precision[]. (in a separate PR)